src/Parser/ConstExprParser.php

```diff
@@ -61,10 +61,18 @@ public function parse(TokenIterator $tokens, bool $trimStrings = false): Ast\Con
 					$tokens->consumeTokenType(Lexer::TOKEN_IDENTIFIER);
 					if ($tokens->tryConsumeTokenType(Lexer::TOKEN_WILDCARD)) {
 						$classConstantName .= '*';
+						if ($tokens->currentTokenType() === Lexer::TOKEN_IDENTIFIER) {
```

Comment:
I feel like this needs a `while` loop - so that we can have multiple wildcards in the name :)

Reply:
OK sure, might as well do a complete solution while I'm at it I guess.

Reply:
Yes, thank you :)